.envrc

```diff
@@ -0,0 +1 @@
+use flake
```

Comment:
I don't think we need this.

Reply:
Quite few nix users use direnv (or nix-direnv) as well which makes entering development environment even nicer experience.

https://github.com/nix-community/nix-direnv

Adding a file with one line is a nice way of adding support for direnv.

Reply:
I think the problem is that some folks have `direnv` and want to use them for some things, but not for others. For instance, I use it for client work to bring env vars into scope. But I don't want to use it for nix shells. I strongly prefer to use `nix develop` for that. Now, whenever I run a command in my clone of this repo, I get the annoying error message that the directory is blocked.

Reply:
I don't think the `.envrc` file should be committed in the repo either. In principle, somebody could want to use a different `.envrc` that doesn't use Nix, but another virtual environment gizmo.

Reply:
The usual thing I saw to use for both cases is to source `.envrc.local` from `.envrc` then you get best of both worlds

Reply:
FYI I use direnv but I wouldn't expect an `.envrc` file to be in the repo but maybe that's my ignorance.

What is the difference between having `use nix` and `use flake`? I and I think @reubenharry are very naive when it comes to nix.

Reply:
> The usual thing I saw to use for both cases is to source `.envrc.local` from `.envrc` (if the file exists ofcourse) then you get best of both worlds

I don't understand the proposed solution. Can you elaborate?

Reply:
Ya, I don't understand why we'd have a stub `.envrc` file and a git ignored `.envrc.local` file. It's really no different than having an ignored `.envrc`, but adds a bit of indirection/complication.